docs/faq.md

```diff
@@ -32,7 +32,23 @@ If Larq helps you in your work or research, it would be great if you can cite it
 }
 ```
 
-If your paper is publicly available, feel free to also add it to the list of [Papers using Larq](https://github.com/larq/docs/edit/master/docs/papers.md).
+If your paper is publicly available, feel free to also add it to the list of papers using Larq below.
+
+## What papers have used Larq?
+
+One of the focuses of Larq is to accelerate research on neural networks with extremely low precision weights and activations.
+Here is a list of papers that have used Larq:
+
+* Helwegen, Koen, et al. ["Latent weights do not exist: Rethinking binarized neural network optimization."](https://papers.nips.cc/paper/8971-latent-weights-do-not-exist-rethinking-binarized-neural-network-optimization.pdf) *Advances in Neural Information Processing Systems (NeurIPS)*. 2019.
+  * Optimization of Binarized Neural Networks (BNNs) currently relies on real-valued latent weights to accumulate small update steps.
```

Comment:
I think you need two more indents here: https://docs-q69010jxj.now.sh/faq/#what-papers-have-used-larq

Reply:
Seems it works now, thanks.